src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java

```diff
@@ -63,7 +63,8 @@ public boolean canEnchantItem(ItemStack item) {
             if (id.isPresent()) {
                 return (id.get().equals(Items.ADVANCED_CIRCUIT.getItemId()))
                     || (id.get().equals(Items.NANO_BLADE.getItemId()))
-                    || (id.get().equals(Items.GLASS_CUTTER.getItemId()));
+                    || (id.get().equals(Items.GLASS_CUTTER.getItemId()))
+                    || (id.get().equals(Items.PORTABLE_CHARGER.getItemId()));
```

Comment:
This is just getting ridiculous at this point, you should probably pass a String Collection in the constructor and check for that rather than all these OR connectors.
Or you get the SlimefunItem for that id and do an instanceof check for a custom interface.

Reply:
If I made a string collection, are there any performance benefits? IMO though this does have a ton of ors, but its much more readable

Reply:
I rather have something effecient then readable

Reply:
I would too, but if there isnt any performance benefit I dont see the point

Reply:
this also looks ugly so i see a point